index.html

```diff
@@ -53,28 +53,38 @@ <h2>Simple example</h2>
 
             <textarea style="width: 100%" id="logs" rows="5"></textarea>
 
-            <script data-live-highlight="simple">
+            <script data-live-highlight="simple" defer>
```

Comment:
@Ofigelov `defer` won't work with inline scripts, why did you add it?

Reply:
My bad, I'll remove it